charts/gardener/operator/templates/clusterrole.yaml

```diff
@@ -142,36 +83,6 @@ rules:
   - rbac.authorization.k8s.io
   resources:
   - clusterroles
-  - roles
   verbs:
   - bind
   - escalate
```

Comment:
Can you add a comment on why these verbs are necessary? This technically gives `gardener-operator` `cluster-admin` permissions if I'm not mistaken. We might want to fix this later on, so it probably deserves a comment.